### ISSUE TITLE
Are the testing cases on project pages generated from texts? 

### ISSUE THREAD

**Issue by TingtingLiao** [bytedance/ImageDream#2]
Hey there! Thanks for your great work. 

I'm curious about the testing cases displayed on the project pages. Are these generated from text prompts or randomly collected from the internet? 

It would be incredibly helpful if you could share some information about it. 

Thank you!

**Comment by raptor-wang** [comment]
Yeah, they are generated using sdxl1.0, and selected by us with background removed, in order to compare with MVDream.

**Comment by TingtingLiao** [comment]
Thanks for your reply. 

My understanding is that these cases have high matched image-text correspondence which make sure the generated results are high quality.  

Have you test ImageDream on real images? Since real images do not have highly accurate text correspondence, I am curious about the performance on real cases or other images not generated from SD.

**Comment by raptor-wang** [comment]
Sure, in this work we majorly focused on generated images, while the training is mostly on real image+ renderred images from objaverse. I think it should also work with real images, with the code you may easily test it on any images you like : )
